testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/config/smallrye/app/MicroProfileConfigTestCase.java

```diff
@@ -57,25 +64,279 @@
     @Deployment
     public static Archive<?> deploy() {
         WebArchive war = ShrinkWrap.create(WebArchive.class, "MicroProfileConfigTestCase.war")
-                .addClasses(TestApplication.class, TestApplication.Resource.class)
-                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+                .addClasses(TestApplication.class)
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml")
+                .addAsManifestResource(MicroProfileConfigTestCase.class.getPackage(),
+                        "microprofile-config.properties", "microprofile-config.properties");
+
+        war.as(ZipExporter.class).exportTo(new File("/tmp", war.getName()), true);
```

Comment:
is there a new to keep this in the test (aside from debugging)?

Reply:
Fixed